test/i18n.api.global.js

```diff
@@ -325,40 +327,6 @@ describe('Module API', function () {
         should.equal(plural, 'Im Baum sitzen 3 Affen')
       })
 
-      it("won't return substitutions when not masked by an extra % (%% issue #49)", function () {
```

Comment:
`printf` by default leaves unbound placeholders in place, i.e. `printf('%1$s')` produces `%1$s`.

I can make this behaviour configurable, though, and restore these tests.

What is preferred here?

* Use the new behavior which makes these tests irrelevant
* Add functionality to fast-printf that adds a way to interpolate unbound placeholders to `undefined` value and restore these tests

Reply:
This can be done now using https://github.com/gajus/fast-printf#handling-unbound-value-references

@mashpie Let me know how you prefer seeing this addressed.